# Opgave I

#### **Spørgsmål I.1 (1)**
Hvilken af følgende udsagn er korrekt?

IQR findes ved at trække Q1 fra Q3. 

In [53]:
# IQR = Q33 - Q1
# Aflæses i blottet.
133-116

[1] 17

Svar: **2**

# Opgave II

#### **Spørgsmål II.1 (2)**
Baseret på tallene i tabellen, hvad er det sædvanlige 95% konfidensinterval for sandsynligheden
for, at hvis du indlægges, er du også indlagt på en intensivafdeling? Antag, at modelantagelserne
er opfyldt.

In [54]:
n = (255+433+153)
p = (62+113+37)/n
# Eller blot
p + c(-1,1) * sqrt(p * (1 - p) / n) * qnorm(0.975)

[1] 0.2227350 0.2814268

Svar: **3**

#### **Spørgsmål II.2 (3)**
For at undersøge udviklingen over tid sammenlignes tallene fra 30. april og 20. marts. Med nul-
hypotesen om at andelen af patienter i ICU kan være ens på de to datoer, hvad er p-værdien
og konklusionen givet signifikansniveau α = 0.05? (Både konklusion og argument skal være
korrekt)

In [55]:
prop.test(x=c(62,37), n=c(255,153), correct=FALSE, conf.level=0.95)


	2-sample test for equality of proportions without continuity correction

data:  c(62, 37) out of c(255, 153)
X-squared = 0.00088915, df = 1, p-value = 0.9762
alternative hypothesis: two.sided
95 percent confidence interval:
 -0.08457431  0.08718868
sample estimates:
   prop 1    prop 2 
0.2431373 0.2418301 


p-værdien findes til 0.9762 hvilket er større end 0.05, hvorfor den vurderes ikke signifikant.

Svar: **3**

#### **Spørgsmål II.3 (4)**
Fordelingen af patienter over forskellige regioner undersøges nu. Tabellen herunder viser antallet
af personer indlagt på hospitaller på forskellige datoer i de 5 regioner i Danmark. Vi antager
her, at den samme person ikke er indlagt på mere end 1 dato.

Under nulhypotesen, hvad er bidraget til teststørrelsen for “Nordjylland ” den 18. marts?

Svar **BLANK**

#### **Spørgsmål II.4 (5)**
Test størrelsen er beregnet til χ2obs = 29. Givet et signifikansniveau α = 0.05, hvad er p-værdien
og konklusionen for den tilsvarende hypotesetest? (Både konklusion og argument skal være
korrekt)

In [56]:
x <- matrix(c(13,33,12,144,53,21,54,35,183,60,32,77,85,251,86,10,16,12,64,27), ncol=5, byrow=TRUE)
chisq.test(x, correct = FALSE)


	Pearson's Chi-squared test

data:  x
X-squared = 29, df = 12, p-value = 0.00394


p-værdien findes til 0.00394 (< alpha), og der er derfor signifikant forskel.

Svar: **5**

#### **Spørgsmål II.5 (6)**
Hvis vi på en given dag antager, at 4% af befolkningen er inficeret med en virus, hvor man-
ge mennesker skal så testes tilfældigt for at få en fejlmargin på maksimalt 1% ved brug af
signifikansniveau α = 0.05?

In [57]:
qnorm(0.975) * sqrt((1/25 * (1-1/25))/1039)
qnorm(0.975) * sqrt((1/25 * (1-1/25))/1476)
qnorm(0.975) * sqrt((1/25 * (1-1/25))/369)
qnorm(0.975) * sqrt((1/25 * (1-1/25))/9603)
qnorm(0.975) * sqrt((1/25 * (1-1/25))/6764)

[1] 0.01191533

[1] 0.009997019

[1] 0.01999404

[1] 0.003919316

[1] 0.004669946

Kan også regnes direkte:

In [58]:
0.04 * 0.96 * (qnorm(0.975)/0.01)^2

[1] 1475.12

Vi skal teste mindst 1476 mennesker for at opnå 1% ME (margin of error).

Svar: **2**

# Opgave III

#### **Spørgsmål III.1 (7)**
En første multipel lineær regressionsmodel blev etableret. Modeloversigten er angivet nedenfor.
Antag, at modelantagelserne er opfyldt!

Sætning 1: Forkert idet begge p-værdier er større end 0.05, og vi kan derfor ikke antage at lønne afhænger heraf. 

Sætning 2: Forkert idet der sammenlignes med 0.025

Sætning 3: Forkert idet vi har tilstrækkelig information.

Sætning 4: Forkert idet begge p-værdier er større end 0.05.

Sætning 5: Rigtig, idet p-værdierne er større end 0.05 hvorfor vi kan vurdere at de ikke har signifikant inflydelse. 

Svar: **5**

#### **Spørgsmål III.2 (8)**
Backwards model selection blev udført for den multiple lineære regressionsmodel ovenfor, hvil-
ket resulterede i følgende R-output:

....

Hvilken R-kode resulterer i det korrekte 95% konfidensinterval for gennemsnittet af Professor
lønnen?

Det må være sætning 3, idet qt anvendes korrekt. 0.975 beskriver netop 95% konf interval, og df = 124 stemmer overens med output nr. 2.

Svar: **3**

# Opgave IV

#### **Spørgsmål IV.1 (9)**
Beregn den teststørrelse, som mangler i modeloversigten ovenfor (manglende værdier er erstat-
tet af spørgsmålstegn i tabellen). Hvilket af følgende svar er rigtigt?

Tester samtlige muligheder:

In [59]:
2 * (1-pt(abs(0.02), df=47-1))
2 * (1-pt(abs(12.25), df=47-1))
2 * (1-pt(abs(0.49), df=47-1))
2 * (1-pt(abs(49.15), df=47-1))
2 * (1-pt(abs(12.49), df=47-1))

[1] 0.9841299

[1] 4.440892e-16

[1] 0.6264629

[1] 0

[1] 2.220446e-16

0 ≃ <2e-16

Svar: **4**

## **Spørgsmål IV.2 (10)**
Vi vil teste hypotesen H0 : β0 = 0, hvor β0 repræsenterer modellens intercept. Hvilket af de
følgende udsagn er korrekt (antag α = 0.05)? (Både argumentation og konklusion skal være
korrekte!)

Udregner først tcrit:

In [60]:
qt(0.975,46)

[1] 2.012896

Idet tobs > tcrit afvises H0.

Svar: **2**

#### **Spørgsmål IV.3 (11)**
I henhold til den lineære model ovenfor, hvad er den forventede stigning i antallet af transistorer
fra 2010 til 2015?

In [61]:
#-6.786e+02 - 3.481e-01 * year
y2010 = -6.786e+02 + 3.481e-01 * 2010
y2015 = -6.786e+02 + 3.481e-01 * 2015

dif = exp(y2015) - exp(y2010)


Svar: **3**

# Opgave V

#### **Spørgsmål V.1 (12)**
Man er interesseret i at bestemme massefylden af en væske. For at gøre dette måles væskens
masse, m, og volumenet, V . Massefylden er angivet ved
m
ρ=
V
Hvad er præcisionen (standardafvigelse, σρ ) for den udregnede massefylde, hvis massen og
volumenet kan måles med en præcision henholdsvis σm = 0.2 og σV = 0.4? Antag at masse- og
volumenmålinger er uafhængige og normalfordelte.

blank

##### **Spørgsmål V.2 (13)**
Lad Xi være en stokastisk variabel. Følgende kode køres i R for at trække 100 tilfældige tal Xi
fra en given fordeling.

In [63]:
x <- rnorm(100)^2 + rnorm(100)^2 + rnorm(100)^2
summary(x)

    Min.  1st Qu.   Median     Mean  3rd Qu.     Max. 
 0.04716  1.33512  2.61460  3.26315  5.14190 10.21328 

Hvilket af følgende udsagn er korrekt?

# Opgave VI

#### **Spørgsmål VI.1 (15)**
Hvilket af følgende er det korrekte 99% konfidensinterval for median-længden af anacondaer,
hvis man antager at parametric bootstrapping blev brugt til at estimere intervallet?

1 er forkert CI interval.

2 er forkert idet vi apply() ved brug af mean. 

3 er rigtig, idet CI interval er korrekt, vi inkluderer mean og sd i rnorm(), og vi apply() ved brug af median. 

Svar: **3**

#### **Spørgsmål VI.2 (16)**
Hvilket af følgende er det korrekte 99% konfidensinterval for median-længden af anacondaer,
hvis man antager at non-parametric bootstrapping blev brugt til at estimere intervallet?

Kan kun være enten 1 eller 2 da det er korrekt CI interval. Tænker det er sample (ud fra bogens eksempel 4.14 der også bruger sample.)

Svar: **1**

# Opgave VII

#### **Spørgsmål VII.1 (17)**
Du har samlet rav med en ven, og I fandt i alt 20 stykker. I havde aftalt på forhånd, at dele
dem ved tilfældigt at trække 10 stykker hver. Tre af stykkerne er meget attraktive. Hvad er
sandsynligheden for, at du får alle tre attraktive stykker?

In [74]:
dhyper(3, m=3, n=20 - 3, k = 10)

[1] 0.1052632

Der er en 10.5% chance.

Svar: **4**

#### **Spørgsmål VII.2 (18)**
Lad X repræsentere vægten i gram af et nyt stykke rav, som du finder på dit foretrukne
samlested. Fra erfaring ved du, at når du finder et stykke rav der, så følger dets vægt en
log-normal distribution, således at X ∼ LN (1, 0.7^2 ).
Hvad er gennemsnitsvægten µX af ravstykker på din favorit placering i henhold til denne model?

Bruger bogens formel (2-65)

In [76]:
exp(1 + 0.7^2/2)

[1] 3.472935

Svar: **3**

#### **Spørgsmål VII.3 (19)**
Baseret på oplysningerne i det sidste spørgsmål: Hvis du finder 20 stykker rav på dit foretrukne
samlested, hvad er sandsynligheden for, at mindst 3 af dem vejer over 10 gram?

In [92]:
(1 - pbinom(2,20, 1 - plnorm(10,1,0.7))) * 100

[1] 2.363236

Der er en 2.36% chance.

Svar: **2**

# Opgave VIII

#### **Spørgsmål VIII.1 (20)**
Stikprøvegennemsnittet er
...

Hvad er fordelingen af X̄ når n går mod uendeligt?

Normalfordelt. mu og sigma findes fra bogens sætning 2.36.

Svar: **4**

#### **Spørgsmål VIII.2 (21)**
1
Definer Yi = 2 + 10
Xi . Hvilket af følgende udsagn er korrekt?

Bruger bogens sætning 2.54:

Y = a*X+b, then:
V(Y) = V(a*X+b)=a^2V(X)

Dermed (1/10)^2 V(X)

Svar: **4**

# Opgave IX

#### **Spørgsmål IX.1 (22)**
Lad µwinter være det gennemsnitlige opreguleringsvolume på vinterdage. Hvis man anvender
et signifikansniveau α = 0.05, hvad bliver da konklusionen for følgende nulhypotese (både
konklusion og argument skal være korrekte)?

Giver sig selv, at når mu_0 = 10 er inden for CI så skal den accepteres.

Svar: **5**

#### **Spørgsmål IX.2 (23)**
Hvad er 99% konfidensintervallet for µwinter ?

Finder først standard error. Kan findes da vi kender et CI, mean og df:

In [95]:
# øvre 95% ci - mu / tobs
(12.341 - 10.84 ) / qt(0.975, df=89)

[1] 0.7554183

In [105]:
mu = 10.84
df = 89
se = 0.755418264107695
mu + c(-1,1) * qt(1 - 0.01/2, df) * se

[1]  8.851585 12.828415

Løsningsforslag:

Udregn 1/2 CI width og udled s ud fra denne.

In [102]:
(12.341 - 9.346)/2 

# tobs
qt(0.975, df=89)

[1] 1.4975

[1] 1.986979

In [109]:
# finding SE:
# SE = s/sqrt(n)
# s/sqrt(n) = 0.5 * CI_width / tobs
1.4975 / 1.98697869950628

[1] 0.7536568

Vi får samme SE og regner dermed CI

In [110]:
mu = 10.84
df = 89
se = 0.753656795803647
mu + c(-1,1) * qt(1 - 0.01/2, df) * se

[1]  8.856221 12.823779

Svar: **4**

#### **Spørgsmål IX.3 (24)**
Hvad er antallet af observationer i xwinter?

udledes ud fra df idet n = df - 1

In [111]:
df = 89
(n = df + 1)

[1] 90

Svar: **3**

#### **Spørgsmål IX.4 (25)**
For at finde ud af, om der er forskel mellem vinter og sommer, indlæses de daglige gennemsnit
af opreguleringsvolume for sommermånederne det samme år i xsummer.
Baseret på de givne data her i opgaven, hvilken af følgende tests er bedst egnet til at konkludere,
om der er en signifikant forskel mellem den daglige middelværdi af opreguleringsvolume om
vinteren og om sommeren?

Giver mening at udføre en two-sample t-test. Sommer og vinter er hver deres tid, så de er ikke udført samtidig og kan derfor ikke være en paired test.

Svar: **1**

# Opgave X

#### **Spørgsmål X.1 (26)**
Ved et signifikansniveau på 5%, hvad er da den kritiske værdi for F -testen for ens middelværdi
på ugedagene?

Bruger qf() da det er kritisk værdi for en F test.

In [116]:
qf(0.95, 6, 357)

[1] 2.123994

Svar: **4**

#### **Spørgsmål X.2 (27)**
Under antagelse af, at alle forudsætninger for den anvendte model er opfyldt, hvad er da esti-
matet af variansen af den daglige nedreguleringspris på fredage (både værdien og forklaringen
skal være korrekt)?